### PR TITLE
Add Tezos RPC methods

### PR DESCRIPTION
**Pull request by Hactive808**

### CODE REVIEW

File: docs/rpc/tezos-rpc-documentation/getBlockHeader.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+```typescript
+// Import required libraries and modules from Tatum SDK
+import { TatumSDK, Tezos, Network } from '@tatumio/tatum';
+
+// Initialize the Tatum SDK for Tezos
+const tatum = await TatumSDK.init<Tezos>({ network: Network.TEZOS });
+
+// Define the chain ID (For this example, we use "NetXdQprcVkpaWU" as a placeholder)
+const params = { 
+    chainId: 'NetXdQprcVkpaWU',
```
Comment (Hathoriel) [inline docs/rpc/tezos-rpc-documentation/getBlockHeader.md]: Why one time you have const params = { 
    chainId: 'YOUR_CHAIN_ID',           // Specify the chain ID (Network identifier)
    blockId: 'YOUR_BLOCK_ID',           // Optional: Specify the block ID (hash)
};

And second time
const params = { 
    chainId: 'NetXdQprcVkpaWU',           // Specify the chain ID (Network identifier)
    blockId: 'BLVRfBFc8Mb7s4PUacDfaeQTPj9HtzTNEArM1Ni1YbxRuK73htp',           // Optional: Specify the block ID (hash)
};

?

**Review by Hathoriel — CHANGES_REQUESTED** [review]

File: docs/rpc/tezos-rpc-documentation/isBootstrapped.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+The `isBootstrapped` method returns an object containing detailed information regarding the node's bootstrap and synchronization status:
+
+- `bootstrapped` (boolean): 
+  Indicates whether the node has completed the bootstrapping process. `true` signifies that the node has bootstrapped, while `false` indicates otherwise.
+
+- `sync_state` (`$chain_status`):
+  Provides the current synchronization status of the node in relation to its peers. The possible values for `$chain_status` are:
+  - `stuck`: The node considers itself synchronized with its peers, but from its perspective, the chain seems halted.
+  - `synced`: The node believes it is in sync with its peers and its current head timestamp is recent.
+<<<<<<< HEAD:docs/rpc/tezos-rpc-documentation/isBootstrapped.md
```
Comment (Hathoriel) [inline docs/rpc/tezos-rpc-documentation/isBootstrapped.md]: here it is wrongly merged

File: docs/rpc/tezos-rpc-documentation/isBootstrapped.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# isBootstrapped
+
+### Hoe to use it
```
Comment (Hathoriel) [inline docs/rpc/tezos-rpc-documentation/isBootstrapped.md]:
```suggestion
### How to use it
```

File: docs/rpc/tezos-rpc-documentation/isBootstrapped.md (hunk 2 above)
Comment (Hactive808) [reply docs/rpc/tezos-rpc-documentation/isBootstrapped.md]: fixed

File: docs/rpc/tezos-rpc-documentation/getBlockHeader.md (hunk 1 above)
Comment (Hactive808) [reply docs/rpc/tezos-rpc-documentation/getBlockHeader.md]: fixed

**Review by Hathoriel — APPROVED** [review]

**Pull request merged**